src/audio/google_rtc_audio_processing.c

```diff
@@ -261,6 +263,12 @@ static int google_rtc_audio_processing_prepare(struct comp_dev *dev)
 
 	cd->output = list_first_item(&dev->bsink_list, struct comp_buffer, source_list);
 
+	if (cd->num_aec_reference_channels > cd->aec_reference->stream.channels) {
```

Comment:
shouldnt this be `!=` ?

Reply:
Yes and no:
On some platform the playback output is left right left right due to a crossover later on on the signal processing chain. That makes the aec_reference be 4 channels and the AEC should only use the 2 first so in that case it should be `>`.
However, I think I want long-term find a solution which would enforce the `!=` but it would be in a follow-up PR

Reply:
Likely a comment is needed here repeating your above statement.

Reply:
Done !